apps/etterlatte-beregning/src/main/kotlin/beregning/grunnlag/BeregningsGrunnlagService.kt

```diff
@@ -76,9 +76,9 @@ class BeregningsGrunnlagService(
 
                 val soeskenMedIBeregning =
                     barnepensjonBeregningsGrunnlag.soeskenMedIBeregning.ifEmpty {
-                        when (val virkningstidspunkt = behandling.virkningstidspunkt) {
+                        when (behandling.virkningstidspunkt) {
                             null -> throw RuntimeException("Kan ikke lagre default soeskenjustering uten virkningstidspunkt")
-                            else -> listOf(GrunnlagMedPeriode(emptyList(), virkningstidspunkt.dato.atDay(1), null))
+                            else -> emptyList()
```

Comment:
I stedet for bare rett frem emptylist burde vel denne se på om virk er etter reform, og hvis det er før reform kaste en badrequest-type feilmelding?

Reply:
Eller vent nå litt, vil du ikke la oss ha beregning hvis det er før reform?

Reply:
https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/3408/commits/14fbbe69731b8bdfc86fecde2f80c6b8f7cc1934